internal/controller/operatorconfigmap_controller.go

```diff
@@ -598,17 +598,15 @@ func (c *OperatorConfigMapReconciler) ensureConsolePlugin() error {
 func (c *OperatorConfigMapReconciler) getNoobaaSubManagementConfig() bool {
 	valAsString, ok := c.operatorConfigMap.Data[manageNoobaaSubKey]
 	if !ok {
-		//while feature in dev preview returning false to disable feature by default.
-		return false
+		return true
 	}
 	val, err := strconv.ParseBool(valAsString)
 	if err != nil {
 		c.log.Error(
 			err,
 			"Unsupported value under manageNoobaaSubscription key",
 		)
-		//while feature in dev preview returning false to disable feature by default.
```

Comment:
Is this not dev-preview anymore? if not please update the commit message for this change as well.

Reply:
the issue was, the feature was incorrectly tied to upgrade which is wrong (mentioned in description). there is no change to feature state.

commit msg vs pr description, since we are doing merge commits, I'm increasingly updating the pr description rather than duplicating at multiple places if there needs to be a change 😅

Reply:
agreed, I follow that for features or if the PR is addressing more than 1 commit.